react/components/PickupPointHour.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { FormattedTime } from 'react-intl'
+
+interface Props {
+  time: string
+}
+
+const PickupPointHour: React.FC<Props> = (props) => {
+  const hoursNumbers = props.time.split(':').map((t) => parseInt(t, 10)) as [
```

Comment:
is `props.time` always guaranteed to be a `XX:XX:XX` string? maybe add a fallback

Reply:
> is `props.time` always guaranteed to be a `XX:XX:XX` string?

I believe so, this is coming from the API and the previous code also relied on this being true